biodatacatalyst-ui/src/main/webapp/picsureui/header/header.js

```diff
@@ -1,44 +1,159 @@
-define(["jquery", "backbone", "handlebars", "studyAccess/studyAccessFunctions", "text!header/header.hbs", "overrides/header", "text!../settings/settings.json"],
-    function($, BB, HBS, studyAccess, template, overrides, settings){
+define(["jquery", "backbone", "handlebars", "text!header/header.hbs", "overrides/header",
+        "text!../settings/settings.json", "text!options/modal.hbs","text!header/userProfile.hbs", "psamaui/overrides/userProfile",
+        "util/notification", "picSure/userFunctions"],
+    function($, BB, HBS, template, overrides,
+             settings, modalTemplate, userProfileTemplate, profileOverride,
```

Comment:
This project is already an override;  I don't think we need to worry about extending the extension.  

If there is some custom BDC code in the psamaui/overrides/userProfile file it should probably replace the default; if there is no code or logic there, then we don't have to worry about checking for it later (line 58)

Reply:
same thing with "overrides/header" - in this repo we control the final setup, so we don't have to worry about splitting out override logic, since everything is an override.

Reply:
Looking at this now, I'm wondering if we even need to override header.js like this...It was only created to override the study access tab, which can now be handled generically I think via the base header.js. We might want/need to do this eventually for open-pic-sure though